### PR TITLE
Adding logs to debug lock refreshing

### PR DESCRIPTION
**Pull request by nimai-agarwal**
## General
Trace level logs that can be enabled to debug how often locks are refreshed

### CODE REVIEW

**Review by LucasIME — APPROVED** [review]

File: lock-api/src/main/java/com/palantir/lock/client/NamespacedConjureTimelockServiceImpl.java [hunk 1]
```diff
@@ -85,12 +89,34 @@ public ConjureWaitForLocksResponse waitForLocks(ConjureLockRequest request) {
...
+                "Attempting to refresh locks V2",
+                SafeArg.of("namespace", namespace),
+                SafeArg.of("allLocksSize", request.get().size()));
+        ConjureRefreshLocksResponseV2 response = conjureTimelockService.refreshLocksV2(AUTH_HEADER, namespace, request);
+        log.trace(
+                "Finished refreshing locks V2",
+                SafeArg.of("namespace", namespace),
+                SafeArg.of("allLocksSize", request.get().size()),
+                SafeArg.of("refreshedLocksSize", response.getRefreshedTokens().size()),
+                SafeArg.of("refreshedAll", request.get().equals(response.getRefreshedTokens())));
```
Comment (jeremyk-91) [inline lock-api/src/main/java/com/palantir/lock/client/NamespacedConjureTimelockServiceImpl.java]: This is good! But I think we should wrap these calls in an `isTraceEnabled`: in a sense, people do sometimes fire off big lock requests, and `equals(...)` I imagine might not be cheap in some cases so we might not want to do it unless trace logging is turned on

**Review by jeremyk-91 — COMMENTED** [review]
Thanks for the contribution! Looks good - one question though

File: lock-api/src/main/java/com/palantir/lock/client/NamespacedConjureTimelockServiceImpl.java (hunk 1 above)
Comment (nimai-agarwal) [reply lock-api/src/main/java/com/palantir/lock/client/NamespacedConjureTimelockServiceImpl.java]: Updated

**Review by jeremyk-91 — APPROVED** [review]
👍

**Pull request merged**